types/is/index.d.ts

```diff
@@ -371,6 +371,11 @@ interface IsStatic {
      */
     ie(value?: number): boolean;
 
+    /**
+     * Checks if current browser is edge.
+     */
+    edge(): boolean;
```

Comment:
Please, add the version parameter. Ref.: [https://github.com/arasatasaygin/is.js#isedgerangenumberstring](https://github.com/arasatasaygin/is.js#isedgerangenumberstring)

Reply:
Added, almost all the other functions were missing this parameter so I've added it for all of them and also expanded the scope of this PR to update the whole environment section